scrapi/harvesters/crossref.py

```diff
@@ -26,13 +26,12 @@
 def process_contributor(author, orcid):
     name = HumanName(author)
     return {
-        'prefix': name.title,
-        'given': name.first,
-        'middle': name.middle,
-        'family': name.last,
-        'suffix': name.suffix,
+        'name': author,
```

Comment:
Any reason why this isn't a helper?
It looks like its used in a couple other places.